docs/user/security/authentication/index.asciidoc

```diff
@@ -397,6 +397,14 @@ NOTE: *Public URL* is available only when anonymous access is configured and you
 +
 For more information, refer to <<embedding, Embed {kib} content in a web page>>.
 
+[float]
```

Comment:
__note:__ I initially planned to document invalidate API in the follow-up and combine it with the slightly related small change we were talking about [here](https://github.com/elastic/kibana/issues/94206#issuecomment-795209440), sooo that's my excuse for this change in this PR :slightly_smiling_face: